### PR TITLE
added example page for TouchableWithoutFeedback

### LINKED ISSUE #257: Add TouchableWithoutFeedback Example page
We don't currently have an example page for `TouchableWithoutFeedback` in gallery... we should add it 😊

### PR DESCRIPTION
**Pull request by YajurG**
### Why

Resolves #257 

### What

Added a new page with example use cases and a select option for the page in the sidebar

## Screenshots

image

### CODE REVIEW

File: src/examples/TouchableWithoutFeedbackExamplePage.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+import {TouchableWithoutFeedback} from 'react-native-windows';
+import {Banner} from '@react-native-material/core';
+
+export const TouchableWithoutFeedbackExamplePage: React.FunctionComponent<{}> = () => {
+  const [title, setTitle] = useState(0);
+  const [focus, setFocus] = useState(false);
+  const {colors} = useTheme();
+
+  const example1jsx = `<TouchableWithoutFeedback
+    >
```
Comment (chiaramooney) [inline src/examples/TouchableWithoutFeedbackExamplePage.tsx:16]: nit: move this > to the end of line 15

Comment (YajurG) [reply src/examples/TouchableWithoutFeedbackExamplePage.tsx:16]: Okay - there's a few instances of this I'll fix them all and re-push

File: src/examples/TouchableWithoutFeedbackExamplePage.tsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    <Text style={{color: colors.text}}>TouchableWithoutFeedback</Text>
+  </TouchableWithoutFeedback>`;
+
+  const example2jsx = `<TouchableWithoutFeedback
```
Comment (chiaramooney) [inline src/examples/TouchableWithoutFeedbackExamplePage.tsx:20]: nit: you'll want the formatting of these jsx snippets to match the formatting of the jsx itself. This way all of our code snippets are formatted the same.

 ">" tags should not be on their own line and make sure the tabbing is correct (i.e. only the child element should be indented inwards)

File: src/examples/TouchableWithoutFeedbackExamplePage.tsx [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    accessibilityValue={{text: ${title}}}
+    onPress={() => {
+        setTitle(title + 1);
+    }}
+    >
+    <Text style={{color: colors.text}}>{String(title)}</Text>
+    </TouchableWithoutFeedback>`;
+
+  const pressableMsg =
+    'Note: it is strongly recommended to use the Pressable API instead of TouchableWithoutFeedback';
```
Comment (chiaramooney) [inline src/examples/TouchableWithoutFeedbackExamplePage.tsx:46]: nit: capitalize "it" and add a period to the end of the sentence.

File: src/examples/TouchableWithoutFeedbackExamplePage.tsx [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+  return (
+    <Page
+      title="Touchable Without Feedback"
+      description="A View that does not provide feedback on touch"
```
Comment (chiaramooney) [inline src/examples/TouchableWithoutFeedbackExamplePage.tsx:51]: nit: add period to end of sentence.

**Review by chiaramooney — COMMENTED** [review]
Looks good just a couple of formatting bits to fix. Could you also add a second screenshot showing the remaining examples not captured in the first screenshot?

**Comment by YajurG** [comment]
Second screenshot of examples:
image

**Review by TatianaKapos — APPROVED** [review]
LGTM! Feel free to hit resolved on the comments you've fixed, it makes it easier for the next person reviewing the PR to look at.

**Pull request merged**